ui/pages/password-reset/index.js

```diff
@@ -99,7 +91,10 @@ export default function PasswordReset() {
                   />
                 </div>
                 <button
-                  disabled={!email}
+                  disabled={submitted}
+                  onClick={() => {
+                    setSubmitted(true)
```

Comment:
Thanks for fixing!

This should be set `onSubmit` handler above (vs writing this new handler)